README.md

```diff
@@ -1,70 +1,150 @@
-# Getting Started with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Available Scripts
+# Math magicians
```

Comment:
- [ ] ❗ Kindly correct the title of your application above. It should read as something like Bookstore and not Math-magicians.

Reply:
Thank you very much